### PR TITLE
[darwin] Add support for Request/Response streaming

### PR DESCRIPTION
**Pull request by stffabi**

### CODE REVIEW

File: website/docs/reference/options.mdx [hunk 1]
```diff
@@ -258,11 +258,11 @@ Not all features of an `http.Request` are currently supported, please see the fo
 | DELETE                  | ✅  | ✅  | ✅ [^1] |
 | Request Headers         | ✅  | ✅  | ✅ [^1] |
 | Request Body            | ✅  | ✅  | ❌      |
-| Request Body Streaming  | ❌  | ❌  | ❌      |
+| Request Body Streaming  | ✅  | ✅  | ❌      |
```
Comment (leaanthony) [inline website/docs/reference/options.mdx:261]: Was Windows updated as well?

**Review by leaanthony — APPROVED** [review]
Awesome work!

File: website/docs/reference/options.mdx (hunk 1 above)
Comment (stffabi) [reply website/docs/reference/options.mdx:261]: We already supported that on Windows for quite some time. Just the docs wasn't updated 😄

Comment (stffabi) [reply website/docs/reference/options.mdx:261]: Response Streaming for Windows will come soon 😄

**Comment by stffabi** [comment]
Thanks for approving 🙏

**Pull request merged**